src/components/Gesture.svelte

```diff
@@ -95,6 +98,9 @@
     }
   }
 
+  $: isGestureRenamed = $nameBind !== defaultNewName;
+  $: showAddActionWalkThrough = !isGestureRenamed && showWalkThrough;
```

Comment:
Assumption that users will not call their action the default name ("Name of action")